### YamlMime:PythonClass
uid: azure.mgmt.dataprotection.models.CloudError
name: CloudError
fullName: azure.mgmt.dataprotection.models.CloudError
module: azure.mgmt.dataprotection.models
summary: An error response from Azure Backup.
constructor:
  syntax: 'CloudError(*args: Any, **kwargs: Any)'
variables:
- description: The resource management error response.
  name: error
  types:
  - <xref:azure.mgmt.dataprotection.models.Error>
attributes:
- uid: azure.mgmt.dataprotection.models.CloudError.error
  name: error
  summary: The resource management error response.
  signature: 'error: _models.Error | None'
